scala-package/core/pom.xml

```diff
@@ -13,6 +13,10 @@
   <artifactId>mxnet-core</artifactId>
   <name>MXNet Scala Package - Core</name>
 
+  <properties>
+    <skipTests>false</skipTests>
```

Comment:
I think the skipTests property is already used to refer to skipping the scala integrationtests. Maybe rename this to runsJavaTests or something

Reply:
Good point. Done in 0be5765d6d65412b85f8479c33f01ad68713fa69